hudi-cli/pom.xml

```diff
@@ -122,6 +122,31 @@
           <includeTestSourceDirectory>false</includeTestSourceDirectory>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <version>2.22.0</version>
+        <configuration>
+          <includes>
+            <include>**/ITT*.java</include>
+          </includes>
+        </configuration>
+        <executions>
+          <execution>
+            <phase>integration-test</phase>
```

Comment:
> Are you sure we must introduce this `phase` in `hudi-cli` module?

This line is non-essential, it can indicate that the binding is `integration-test`. Maybe let it here is also ok. if need remove, I will delete it.

Reply:
I am thinking if we want to bind with `integration-test` phase. Why not move to the integration test module?

Reply:
@yanghua I've thought about it before, this test dependency on spring-shell and  integration-test module can not use `getShell().executeCommand("....")` to access `commands` in hudi-cli.

Reply:
Fine. Let's try to figure out a better way.